enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -105,9 +106,25 @@
    opts are passed to load-metabase"
   [path :- :string
    opts :- [:map
-            [:abort-on-error {:optional true} [:maybe :boolean]]
             [:backfill? {:optional true} [:maybe :boolean]]]]
-  (v2-load-internal! path opts :token-check? true))
+  (let [start    (System/currentTimeMillis)
+        err      (atom nil)
+        report   (try
+                   (v2-load-internal! path opts :token-check? true)
+                   (catch Exception e
+                     (reset! err e)))
+        imported (set (map (comp :model last) (:seen report)))]
+    (snowplow/track-event! ::snowplow/serialization-import nil
+                           {:source        "cli"
+                            :duration      (long (/ (- (System/currentTimeMillis) start) 1000))
+                            :models        (str/join "," imported)
+                            :count         (if (contains? imported "Setting")
+                                             (inc (count (remove #(= "Setting" (:model (first %))) (:seen report))))
```

Comment:
I'm curious about this "remove then count one extra" logic - is it possible that 0 or more than 1 are removed? A small comment could be helpful.

Reply:
Oh, ok! The thing is that every Setting is counted as a separate instance and it felt like it could skew understanding of a size of serialization batch a bit. So this is me accounting for it instead of making everyone analyzing account for settings in their head.

Reply:
I'm still a bit confused though - in practice how is this count different from what you'd get if you just did a direct count?

Reply:
Settings are about 70-80+ keys and they are cheap, so for me it skewed the picture, it went from 5 entities to 90 actually. :) So to keep this number at least somewhat relevant I decided not to count each individual one.